public/docs/_examples/toh-5/dart/lib/heroes_component.dart

```diff
@@ -26,7 +26,11 @@ class HeroesComponent implements OnInit {
   Hero selectedHero;
 
   // #docregion renaming
-  HeroesComponent(this._heroService, this._router);
+  HeroesComponent(this._heroService,
```

Comment:
The trade-off is to enable plaster /* ... */ in the code excerpt, or to leave it as is. IMHO, even if a particular SDK doesn't support it, readers should understand that the code excerpt is suggestive, given that the heading is "_lib/heroes_component.dart (showing renamings only)_". Use of plaster just makes it too cluttered.